pkg/orm/mock_test.go

```diff
@@ -30,24 +31,25 @@ func TestGenEpochMock(t *testing.T) {
 	}
 	require.Equal(t, int64(11), epoch)
 
-	// FIXME: go-mysql-server concurrent transaction may cause data race
-	// need mutex protection
+	// FIXME: 'no such table: logic_epochs' will raise, why??
 	/*
```

Comment:
Can we enable this case now

Reply:
We can't do that now. A new connection(creat when we start a tx) will create a new DB, and cause 'no such table' error consequently.
But if we enable 'cache=shared' option for sqlite, we can solve this problem and cause other problems because we enable 't.Parallel()' and cases will shared the same DB at the mean time.